.github/workflows/build-linux-release.yml

```diff
@@ -40,6 +43,14 @@ jobs:
         TAG: ${{ steps.get_version.outputs.VERSION }}
         BUCKET: ${{ secrets.BUCKET }}
 
+    - name: Create RPM package 
+      run: ./.github/workflows/build-rpm-pkg.sh
+      env:
+        AWS_ACCESS_KEY_ID: ${{ secrets.UPLOAD_ID }}
+        AWS_SECRET_ACCESS_KEY: ${{ secrets.UPLOAD_KEY }}
+        TAG: ${{ steps.get_version.outputs.VERSION }}
+        BUCKET: ${{ secrets.BUCKET }}
```

Comment:
Maybe we would want this to be in a different bucket? Its content will obviously grow with every release, and there is a manual process for devs to get the build and sign it. 

Just asking for your opinion.

Reply:
I don't think we need a different bucket.  But we might want to break up the linux packages into apt and yum sub directories or prefix keys.

Reply:
Right, makes even more sense. Would require a change in this and the other action though as well. Do you want to do that or leave that for later?